index.html

```diff
@@ -216,6 +239,7 @@ <h2><code>PointerEvent</code> interface</h2>
     boolean     isPrimary = false;
     sequence&lt;PointerEvent> coalescedEvents = [];
     sequence&lt;PointerEvent> predictedEvents = [];
+    DeviceProperties deviceProperties;
```

Comment:
Good point! I've made the change to add a regular constructor instead of a constructor + init dictionary since there is only one attribute in `deviceProperties`.

Reply:
I think that we should pass an init dictionary for future compatibility to have additional properties.

Reply:
Done